PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Collection/PlatformInformationCollector.cs

```diff
@@ -22,13 +23,74 @@ namespace Microsoft.PowerShell.CrossCompatibility.Collection
     /// </summary>
     public class PlatformInformationCollector : IDisposable
     {
+        /// <summary>
+        /// Collect all release info files into a lookup table in memory.
+        /// Overrides pre-existing keys if there are duplicates.
+        /// </summary>
+        /// <returns>A dictionary with the keys and values of all the release info files on the machine.</returns>
+        public static IReadOnlyDictionary<string, string> GetLinuxReleaseInfo()
+        {
+            var dict = new Dictionary<string, string>();
+
+            foreach (string path in s_releaseInfoPaths)
+            {
+                try
+                {
+                    using (FileStream fileStream = File.OpenRead(path))
```

Comment:
DRY:
```suggestion
                    using (var fileStream = File.OpenRead(path))
```

Reply:
There's no repetition here; `FileStream` only occurs once on the line and it's not obvious that `File.OpenRead` returns that type